### ISSUE TITLE
Upgrade npmVersion to 5.6.0 on build.gradle to avoid error on Windows

### ISSUE THREAD

**Issue by ayuwono** [marklogic/marklogic-data-hub#804]
On windows OS, when running gradle.bat quick-start:test, this error below is thrown

It's on marklogic-data-hub:npmInstallUInpm task

It seems that it's because of npm version is 5.4.2

When I upgraded npmVersion to 5.6.0 on build.gradle (reference: https://github.com/npm/npm/issues/17671), the task is passing

We should upgrade npmVersion to 5.6.0

```
:marklogic-data-hub:npmInstallUInpm ERR! path C:\jenkins\workspace\Labs\DHF_e2e_QuickStart_qa_b9_nightly_windows\marklogic-data-hub\src\trace-ui\node_modules\fsevents\node_modules
npm ERR! code EPERM
npm ERR! errno -4048
npm ERR! syscall scandir
npm ERR! Error: EPERM: operation not permitted, scandir 'C:\jenkins\workspace\Labs\DHF_e2e_QuickStart_qa_b9_nightly_windows\marklogic-data-hub\src\trace-ui\node_modules\fsevents\node_modules'
npm ERR!  { Error: EPERM: operation not permitted, scandir 'C:\jenkins\workspace\Labs\DHF_e2e_QuickStart_qa_b9_nightly_windows\marklogic-data-hub\src\trace-ui\node_modules\fsevents\node_modules'
npm ERR!   stack: 'Error: EPERM: operation not permitted, scandir \'C:\\jenkins\\workspace\\Labs\\DHF_e2e_QuickStart_qa_b9_nightly_windows\\marklogic-data-hub\\src\\trace-ui\\node_modules\\fsevents\\node_modules\'',
npm ERR!   errno: -4048,
npm ERR!   code: 'EPERM',
npm ERR!   syscall: 'scandir',
npm ERR!   path: 'C:\\jenkins\\workspace\\Labs\\DHF_e2e_QuickStart_qa_b9_nightly_windows\\marklogic-data-hub\\src\\trace-ui\\node_modules\\fsevents\\node_modules' }
npm ERR! 
npm ERR! Please try running this command again as root/Administrator.
```

┆Status: Rejected

**Comment by aebadirad** [comment]
I worked this into the merge for #805 please confirm it addresses the issue and close if so.

**Comment by aebadirad** [comment]
Fixed with release of 3.0.0
